src/pages/home/report/ReportActionsView.js

```diff
@@ -96,7 +193,35 @@ function ReportActionsView(props) {
 
     const isFocused = useIsFocused();
     const reportID = props.report.reportID;
-    const hasNewestReportAction = lodashGet(props.reportActions[0], 'isNewestReportAction');
+
+    /**
+     * Retrieves the next set of report actions for the chat once we are nearing the end of what we are currently
+     * displaying.
+     */
+    const throttledLoadNewerChats = useCallback(
+        () => {
+            if (props.isLoadingNewerReportActions || props.isLoadingInitialReportActions) {
+                return;
+            }
+
+            // eslint-disable-next-line no-use-before-define
```

Comment:
We definitely should not have to suppress `no-use-before-define`

Reply:
I'll restructure it and remove these suppressors.